### ISSUE TITLE
Create private tesntet example using kind

### ISSUE THREAD

**Issue by skylenet** [skylenet/ethereum-k8s-testnets#3]
Currently we only have an example using a kubernetes cluster provided by Digitalocean.

It would be good to also provide an example of a private testnet that can run fully at the local host. To make this work, we could use [kind](https://kind.sigs.k8s.io/)

**Comment by muddlebee** [comment]
@skylenet  

I am planning on creating a private testnet using https://www.vultr.com/ which provides super cheap instances.

Vultr supports Terraform. https://registry.terraform.io/providers/vultr/vultr/latest/docs

I hope I am going in the right direction. Anything else that I need to cater along with this? Please provide your inputs/suggestions 😄

**Comment by skylenet** [comment]
> I hope I am going in the right direction. Anything else that I need to cater along with this? Please provide your inputs/suggestions smile

Yes, once you have your kubernetes cluster setup on Vultr you should be able to reuse most of the helmsman code. The only things that could change is how storage is used. I didn't hardcode any digitalocean specific storage class, which means that it should pick up the default storage class on the Vultr k8s cluster.
